app/views/content/life-as-a-teacher/teaching-as-a-career/from-qualified-teacher-to-head-of-biology/_article.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+      <h2>I knew teaching was for me</h2>
+      <p>From an early age, I knew a teaching career was for me. I could not imagine working a typical nine-to-five desk job. Every day in teaching brings a new challenge and new situations, which is what makes the job so exciting.</p>
+      <p>When your pupils have those ‘light bulb’ moments, hearing the sudden ‘So that’s how it works!’ and seeing them take your subject into A level and beyond makes the job so worthwhile.</p>
```

Comment:
```suggestion
      <p>When your pupils have those lightbulb moments, hearing the sudden ‘So that’s how it works!’ and seeing them take your subject into A level and beyond makes the job so worthwhile.</p>
```
we changed Peter's story to have lightbulb as one word!

also - do we really need the quote marks round it? we didn't in Peter's story